crates/model/src/order.rs

```diff
@@ -660,6 +666,9 @@ pub struct OrderMetadata {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub onchain_user: Option<H160>,
     pub is_liquidity_order: bool,
+    /// Full app data that `OrderData::app_data` is a hash off. Can be None if
+    /// the backend doesn't know about the full app data.
```

Comment:
Might be nice to explain what "the backend doesn't know about the full app data" is useful for any where this app data typically is. (It's typically on IPFS, right?)